assets/js/hooks/Mapper/components/ui-kit/MarkdownTextViewer.module.scss

```
@@ -1,28 +1,36 @@
 .MarkdownTextViewer {
   @apply text-[12px] leading-[1.2] text-stone-300 break-words;
+  font-size: var(--rf-markdown-font-size, 12px);
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)